types/graphql-resolvers/index.d.ts

```diff
@@ -1,37 +1,38 @@
 // Type definitions for graphql-resolvers 0.2
 // Project: https://github.com/lucasconstantino/graphql-resolvers#readme
-// Definitions by: Mike Engel <https://github.com/mike-engel>
+// Definitions by: Mike Engel <https://github.com/mike-engel>, Alejandro Corredor <https://github.com/aecorredor>
+
```

Comment:
Looks like you added an extra line here which might be breaking the header section. Additionally, looking at the [mongodb](https://github.com/smeijer/DefinitelyTyped/blob/a5f93e601f02d025a48ffb2dabd93b4bd9163296/types/mongodb/index.d.ts#L4) types, it looks like maybe each contributor should be on their own line.